packages/node/src/integrations/local-variables/local-variables-sync.ts

```diff
@@ -12,7 +12,29 @@ import type {
   RateLimitIncrement,
   Variables,
 } from './common';
-import { createRateLimiter, functionNamesMatch, hashFrames, hashFromStack } from './common';
+import { createRateLimiter, functionNamesMatch } from './common';
+
+/** Creates a unique hash from stack frames */
+export function hashFrames(frames: StackFrame[] | undefined): string | undefined {
+  if (frames === undefined) {
+    return;
+  }
+
+  // Only hash the 10 most recent frames (ie. the last 10)
+  return frames.slice(-10).reduce((acc, frame) => `${acc},${frame.function},${frame.lineno},${frame.colno}`, '');
```

Comment:
I wonder if we should filter out system frames here, so basically either

1. pick all the frames (if less than 10)
2. pick 10 most recent frames (after filtering out system frames)